app/controllers/idv/in_person/verify_info_controller.rb

```diff
@@ -51,11 +51,7 @@ def invalid_state?
       end
 
       def prev_url
-        if IdentityConfig.store.in_person_ssn_info_controller_enabled
-          idv_in_person_proofing_ssn_url
-        else
-          idv_in_person_step_url(step: :ssn)
-        end
+        idv_in_person_ssn_url
```

Comment:
I think it might be better to merge this PR and then change the url in a separate PR. It's worth thinking through whether this will work though.

Reply:
we're going to follow this suggestion; added a [ticket](https://cm-jira.usa.gov/browse/LG-10586)

Reply:
Make sure to update your testing plan. If a different issue will handle changing the url, the following is no longer accurate. 
```
 Make sure url is in_person/ssn
 Try updating url to in_person_proofing/ssn, it should display a 404
 ```

Reply:
When you do change the url, `in_person_proofing/ssn` should redirect to `in_person/ssn` rather than 404 so that the change is invisible to the user.